.agents/skills/add-locale-batch/scripts/locale_batch_status.py

```
@@ -0,0 +1,121 @@
+from __future__ import annotations
+
+import argparse
+import json
+import subprocess
+import sys
+from pathlib import Path
+from typing import Any
+
+
+def run_json(args: list[str]) -> Any:
+    completed = subprocess.run(args, text=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)